src/query.ts

```diff
@@ -40,22 +49,35 @@ export const getLatestQuery = <T extends WithOrdinal>(index: string): any => ({
   },
 });
 
-export const getMultiQuery = <T, K extends keyof T, S extends keyof T>(
+const isSearchSince = <T>(options: any): options is SortOptionSince<T>[] => {
+  return options.length > 0 && (typeof options[0].searchSince === 'string' || typeof options[0].searchSince === 'number');
+};
+
+const getSearchSince = <T>(sort: SortOptions<T>) => {
+  return sort.options.length == 0
+    ? {}
+    : !isSearchSince<T>(sort.options)
+      ? {}
+      : { search_after: sort.options.map(a => a.searchSince) };
```

Comment:
Can be simplified because you have 2 fallbacks to `{}`
```typescript
sort.options.length !== 0 && isSearchSince<T>(sort.options)
   ? { search_after: sort.options.map(a => a.searchSince) }
   : {}
```